rootfs/standard/var/www/mynode/templates/bitcoin.html

```diff
@@ -20,6 +20,20 @@
                         showAlertPopup("alert_popup", "<b>Password copy failed!</b>");
                     });
                 });
+                $('#bip158_checkbox').change(function () {
+                    $("#bip158_save").show();
+                });
+                $("#bip158_save").on("click", function() {
+                    enabled=$('#bip158_checkbox').is(":checked")
+                    if (enabled)
+                    {
+                        window.location.href='/bitcoin/toggle_bip158?enabled=1'
```

Comment:
Since this takes some extra disk space and may cause Bitcoin to need to sync/process additional data for several hours, could you add a prompt here explaining that? I think it would only need to be shown in the enable case.

A sample can be found on main.html#57 and the other JS functions in mynode.js.

Reply:
`openConfirmDialog()` isn't working on bitcoin page but works on other pages. What am I missing?

Reply:
You probably need the empty, hidden confirm dialog div added at the bottom of the page. Something like:

<div id="confirm-dialog"></div>

The homepage should have an example.

Reply:
Found it and added the confirm dialog